mongo/integration/mtest/mongotest.go

```diff
@@ -873,3 +874,8 @@ func (t *T) interfaceToInt32(i interface{}) (int32, error) {
 
 	return 0, fmt.Errorf("type %T cannot be converted to int32", i)
 }
+
+// GetCSFLEPath returns the test csfle library path provided by the CSFLE_PATH environment variable.
+func GetCSFLEPath() string {
```

Comment:
I haven't seen functions like this for other environment variables (usually we just call `os.Getenv` directly), but this seems like a good pattern to me.

Reply:
I actually ended up removing this because the naming is more wonky after the "csfle" -> "crypt_shared" rename and expanding the `mtest` public API isn't worth the small simplification.